### DISCUSSION TITLE
On production , artisan does not recognize the command - php artisan backup:run

### DISCUSSION THREAD

**Discussion by murilolivorato** [spatie/laravel-backup#1379] [Q&A]
Hello , when I try to run te command on production  -

```
php artisan backup:run

```
it says  -
```
There are no commands defined in the "backup" namespace.
```

I tried to run this command on production -

```
sudo php artisan config:cache
sudo php artisan cache:clear
sudo composer dump-autoload
```
but the artisan still not recognize the laravel backup . 
I checked my  composer.json file in the production and laral backup is there   -

```
 "require": {
        "php": "^7.2.5",
          ....
        "spatie/laravel-backup": "^6.16",
}
```

**Comment by PaolaRuby** [comment]
composer update?

**Comment by MimisK13** [comment]
Same problem here. Any solution ?

**Comment by justinkekeocha** [comment]
I solved this by running compose dump-autoload on production. I am guessing this problem is caused by add —no-scripts in deployment process when running composer install.
